### PR TITLE
cgroup: add support for memory.zswap.current

### PR DESCRIPTION
**Pull request by Flowdalic**

### CODE REVIEW

**Comment by yuwata** [comment]
Maybe, src/run/run.c should be also updated?

**Comment by Flowdalic** [comment]
Updated as in "show the value after the process exits if --wait was given"? That does not seen sensible for an attribute like memory.swap.current (compared to, say, memory.peak).

File: src/systemctl/systemctl-show.c [hunk 1]
```diff
@@ -705,11 +706,14 @@ static void print_status_info(
         if (i->memory_current != UINT64_MAX) {
                 printf("     Memory: %s", FORMAT_BYTES(i->memory_current));
 
+                bool show_memory_swap, show_memory_zswap_current;
```
Comment (YHNdnzj) [inline src/systemctl/systemctl-show.c]: It's not necessary to split the initialization from declaration?

**Comment by yuwata** [comment]
Ah, indeed.

**Comment by yuwata** [comment]
So, there is no memory.zswap.peak ?

**Review by YHNdnzj — APPROVED** [review]
While at it, could you also merge the two show_memory_swap if blocks in systemctl-show.c? (Overlooked that in the review of your previous PR)

**Review by yuwata — APPROVED** [review]

File: src/systemctl/systemctl-show.c (hunk 1 above)
Comment (Flowdalic) [reply src/systemctl/systemctl-show.c]: I somehow as under the impression that this style was expected. Changed.

**Comment by Flowdalic** [comment]
> So, there is no memory.zswap.peak ?
There does not seem to be one.

File: man/org.freedesktop.systemd1.xml [hunk 2]
```diff
@@ -2782,6 +2782,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
       readonly t MemorySwapPeak = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("false")
+      readonly t MemoryZswapCurrent = ...;
```
Comment (poettering) [inline man/org.freedesktop.systemd1.xml:2785]: we so far spelled this as "ZSwap" rather than "Zswap". Compare `MemoryZSwapMax`

Comment (yuwata) [reply man/org.freedesktop.systemd1.xml:2785]: -> #29999.

Comment (Flowdalic) [reply man/org.freedesktop.systemd1.xml:2785]: Darn. I actually looked up how upstream writes it and went with that, while forgetting to look for prior-art in systemd.

**Pull request merged**